config/jobs/bazelbuild/rules_k8s/rules_k8s_config.yaml

```diff
@@ -1,7 +1,7 @@
 presubmits:
   bazelbuild/rules_k8s:
   - name: pull-rules-k8s-e2e
-    cluster: gke_rules-k8s_us-central1-f_testing
+    cluster: gke_rules-k8s_us-west1-b_testing
```

Comment:
I think we want to keep this

Reply:
hmm, maybe not? if we change the zone the name might be misleading

Reply:
/shrug hmm, although the cluster name would now be misleading .

Reply:
I think we have an existing build cluster running on central1-f, and have the cluster config saved as a prow secret, so this job is still going to work..

Reply:
Sen is correct